### DISCUSSION TITLE
Budget optimisation - Both values should be almost equal error

### DISCUSSION THREAD

**Discussion by AruunnAnanda** [google/lightweight_mmm#120] [Q&A] [answered]
Hi there,

I'm running the LMMM Google colab demo file and I get the below error . 
 
![image](https://user-images.githubusercontent.com/107944288/207269445-a91d76d1-e2b7-4e7b-b973-d85e3b1a20b8.png)

Can someone please let me know what should I do here? I don't see any solution.x function for the object "solution". I only see count and index functions for the object "solution". 

![image](https://user-images.githubusercontent.com/107944288/207270096-8e4398fa-4085-4190-b049-659a373b9240.png)

Thanks!

**Comment by statm3n** [comment]
Hi, @AruunnAnanda,

Please refer to  
https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html#scipy.optimize.OptimizeResult
https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.sum.html

And check your input.

**Comment by AruunnAnanda** [comment, marked as answer]
I tried **solution['x']** instead of **solution.x** and it works. Thanks!
